src/tree/kat_encryption.rs

```diff
@@ -253,6 +261,7 @@ fn generate_test_vectors() {
         let mut group = group(ciphersuite);
         *group.epoch_secrets_mut().sender_data_secret_mut() =
             SenderDataSecret::from(sender_data_secret_bytes.as_slice());
+        *group.secret_tree_mut() = group_secret_tree;
```

Comment:
Ugh, this should probably be a `set_secret_tree` function. But looks like it's only used in tests. That's fine.